ls1intum/ArTEMiS
# Enhancement/course overview

### Checklist

### Description
Some UI improvements for the course overview exercise list

### Steps for Testing

1. Log in to ArTEMiS
2. Go to course overview and select a course
3. View the exercise list on a small screen
4. check if everything is finde
